cpp/src/Ice/Makefile.mk

```diff
@@ -28,7 +28,8 @@ endif
 endif
 
 Ice[iphoneos]_excludes                  := $(wildcard src/Ice/CtrlCHandler.cpp $(addprefix $(currentdir)/,Tcp*.cpp Service.cpp))
-Ice[iphoneos]_extra_sources             := $(wildcard $(addprefix $(currentdir)/ios/,*.cpp *.mm))
+Ice[iphoneos]_extra_sources             := $(wildcard $(addprefix $(currentdir)/ios/,*.cpp *.mm))\
+                                           $(wildcard $(addprefix $(currentdir)/iAP/,*.cpp *.mm))
```

Comment:
I think it would be cleaner to add an iAP prefix to all iAP source files and then move them to the ios folder. It's odd to have src/Ice/ios and src/Ice/iAP that are both ios-only.